dev-libs/websocketpp/websocketpp-0.8.1.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Websocket++ header-only library"
+DESCRIPTION="WebSocket++ is a header only C++ library that implements RFC6455 \
+The WebSocket Protocol. It allows integrating WebSocket client and server \
+functionality into C++ programs. It uses interchangeable network transport \
+modules including one based on raw char buffers, one based on C++ iostreams, \
+and one based on Asio (either via Boost or standalone). End users can write \
+additional transport policies to support other networking or event libraries as needed."
+HOMEPAGE="https://github.com/zaphoyd/websocketpp"
+COPYRIGHT="2014, Peter Thorson."
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/zaphoyd/websocketpp/archive/0.8.1.zip"
+CHECKSUM_SHA256="80f49469f1702ec64ef5d8e71302f6b781dca8ae06e560421cab42961c8c7ce6"
+PATCHES="websocketpp-0.8.1.patch"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="websocketpp$secondaryArchSuffix = $portVersion"
+
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:g++$secondaryArchSuffix
+	cmd:make
```

Comment:
These cmd's are only used when building the test items, so they could go into TEST_REQUIRES?

Reply:
With my local changes to the recipe they can stay in :)
https://github.com/Begasus/haikuports/commit/27bb21113ecd6e3abdf98dac6f9ce197cc6f1b16 (latest commit so far)